riot/riot.py

```diff
@@ -156,7 +156,8 @@ def path(self) -> str:
     def venv_path(self) -> str:
         """Return the path to the virtual environment for this interpreter."""
         version = self.version().replace(".", "")
-        return os.path.abspath(f".riot/venv_py{version}")
+        env_base_path = os.environ.get("RIOT_ENV_BASE_PATH") or ".riot/venv_py"
```

Comment:
do we care to keep the `venv_py` prefix as well?

e.g. `RIOT_ENV_BASE_PATH` default is `.riot` only?

Reply:
My thoughts were "if you can edit the path, why not edit ALL THE PATH?" :sweat_smile: 

:+1: I change it

Reply:
yeah, maybe it is weird that we set the virtual env name prefix in the same setting?

Reply:
My ambition was my sin, you're right :upside_down_face: code updated :+1: